x-pack/plugins/security_solution/public/common/components/url_state/constants.ts

```diff
@@ -43,4 +43,6 @@ export type UrlStateType =
   | 'timeline'
   | 'explore'
   | 'dashboards'
-  | 'threat_intelligence';
+  | 'threat_intelligence'
+  | 'cloud_posture'
+  | 'findings';
```

Comment:
could you clarify why only 2 pages are used here?

Reply:
There are some features that depend on URL key but it is not crucial for navigation (administration pages have some unique logic). Didn't find any other effect for this but just followed the pattern of other pages